.markdownlint.yaml

```diff
@@ -1,2 +1,4 @@
 ---
-MD013: false
+# Markdown linting rules to ignore.
```

Comment:
Feels like you've done this to trigger me...

Given that you can do other things besides ignoring [in this file](https://github.com/DavidAnson/markdownlint?tab=readme-ov-file#linting). I think we should remove this comment.

```suggestion
```

Reply:
But everything in here is a rule to ignore.

>  you can do other things

We're not doing other things.

Reply:
But we might?

Reply:
Then we would add another comment above the block of other things?

Reply:
Feels unnecessary, but okay